### ISSUE TITLE
Development halt?

### ISSUE THREAD

**Issue by ngodber** [gmggroup/omf-python#121]
Hi, this is important and much needed project in the geoscience and mining space. I'm disappointed to see that it appears development has been halted? Is there a plan to continue development by Seequent or others? If so, perhaps it is worth shifting focus to adoption of the GEOH5 standard which appears to be well resourced by Mira https://geoh5py.readthedocs.io/en/stable/content/geoh5_format/index.html

**Comment by timmclennan** [comment]
> Hi, this is important and much needed project in the geoscience and mining space. I'm disappointed to see that it appears development has been halted? Is there a plan to continue development by Seequent or others? If so, perhaps it is worth shifting focus to adoption of the GEOH5 standard which appears to be well resourced by Mira https://geoh5py.readthedocs.io/en/stable/content/geoh5_format/index.html

There is work happening on the next version.  In fact, there is a meeting next month to finalise what is in V2.  Details on that meeting can be found at https://www.eventbrite.ca/e/open-mining-format-v20-finalization-workshop-tickets-441189518727.

**Comment by ngodber** [comment]
That is great news, can't wait for v2.0 to launch. I hope you appreciate how it appeared from the outside.

It would be great if the OMF committee could reach out Mira Geoscience to solicit their engagement.  It is a prominent gap in the breadth of support across the industry. @domfournier perhaps Steph or Glenn in BNE could attend?

**Comment by domfournier** [comment]
Thanks for flagging this @ngodber . We are actively working on creating the i/o between geoh5 and omf v0.9.0 so users can export in either format. 

https://github.com/MiraGeoscience/omf/pull/7

We will PR back here when we are done, or hold until v2 is offical then update. Should be straight forward.

Looking forward for v2 release.

**Comment by ngodber** [comment]
That is great news. For clarity it will be 'i' as well as the 'o'?

**Comment by domfournier** [comment]
Correct, the omf -> geoh5 is done, now working on the geoh5 -> omf. At least for the 5 objects that were supported in 0.9.0 (points, surface, curves, grid2D and block models)

**Comment by ngodber** [comment]
Awesome - that's great to hear. That code is going to get a lot of leg work around here.

**Comment by ngodber** [comment]
hi, we are now nearly two years down the road and (seemingly) in the same place? When is the planned launch of 2.0? Will the specification repo be updated with formal spec doc vs inference from python code?

**Comment by mikestewartseeq** [comment]
Hi @ngodber 
There is a steering committee meeting to discuss feedback/endorse OMF V2.0 this coming Wednesday (NZST/AEST).
Expect an announcement soon after.

**Comment by ngodber** [comment]
Great news! Would I be correct in thinking the spec doc will go into the aforementioned github repo?

**Comment by mikestewartseeq** [comment]
That is the logical place to put it. Documentation is one of the agenda items.

**Comment by elphick** [comment]
Hi @mikestewartseeq - Is there any update that can be provided on the likely release date of V2.0?  Maybe I missed the announcement?

**Comment by ThomasMGeo** [comment]
Any update to this? Looks like I can get version 2.0.0a0 using -pre when installing it with pip.
